modules/rpc/yarpc_options.go

```diff
@@ -71,3 +73,20 @@ func onewayInboundMiddlewareFromCreateInfo(mci service.ModuleCreateInfo) []middl
 	// Intentionally panic if programmer adds non-middleware slice to the data
 	return items.([]middleware.OnewayInbound)
 }
+
+func withGraph(graph dig.Graph) modules.Option {
```

Comment:
I've added some thoughts to the dig issue on idiomatic usage. https://github.com/uber-go/fx/issues/285

I think it might be better to start out with more constrained usage patterns until we get more familiar with dig.